pytorch/pytorch_edgeml/graph/rnn.py

```diff
@@ -1050,7 +1050,7 @@ def forward(self, input, hiddenState=None, cellState=None):
 
 class SRNN2(nn.Module):
 
-    def __init__(self, inputDim, outputDim, hiddenDim0, hiddenDim1, cellType):
+    def __init__(self, inputDim, outputDim, hiddenDim0, hiddenDim1, cellType, **fastCellArgs):
```

Comment:
This should be a generic `cellArgs` so for any cell that might be used to day or in the future. Also, make it a dictionary not optional arguments.

Reply:
Ok